### ISSUE TITLE
BridgeBase doesn't compile with Java 11

### ISSUE THREAD

**Issue by ebourg** [eclipse-ee4j/orb-gmbal-pfl#7]
BridgeBase doesn't compile with Java 11 because it uses the now removed Unsafe.defineClass() method:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project pfl-basic: Compilation failure
[ERROR] ~/gmbal-pfl/pfl-basic/src/main/java/org/glassfish/pfl/basic/reflection/BridgeBase.java:[267,22] cannot find symbol
[ERROR]   symbol:   method defineClass(java.lang.String,byte[],int,int,java.lang.ClassLoader,<nulltype>)
[ERROR]   location: variable unsafe of type sun.misc.Unsafe
```

**Comment by russgold** [comment]
I have a PR which should address this by using an earlier JDK for that method, and providing a different implementation for it in the MR jar.

**Comment by russgold** [comment]
Fixed by commit 2124d8a43c4f7cf33
